README.md

```diff
@@ -341,6 +341,34 @@ monday.execute('openFilesDialog', {
 })
 ```
 
+
+#### Trigger file upload process
+Triggers the file upload flow to a specific file column. Rejects the promise in case of error.
+
+After file will be successfully uploaded - "change_column_value" event will be triggered.
```

Comment:
After the file is successfully uploaded, the "change_column_value" event will be triggered. See the [`monday.listen('events', callback)`](https://github.com/mondaycom/monday-sdk-js#mondaylistentypeortypes-callback-params--) method to subscribe to these events.